ci/fetch-rust-docker.bash

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 script_dir=$(cd "$(dirname "$0")" && pwd)
+# shellcheck source=/dev/null
```

Comment:
this should just work, but if its not, 
`# shellcheck source=ci/shared.bash` should

Reply:
Yes, it works. Thanks!